NitroxModel/DataStructures/GameLogic/Entities/WorldEntity.cs

```diff
@@ -14,10 +14,10 @@ namespace NitroxModel.DataStructures.GameLogic.Entities
      */
     [Serializable]
     [DataContract]
-    [ProtoInclude(10, typeof(PlaceholderGroupWorldEntity))]
-    [ProtoInclude(20, typeof(EscapePodWorldEntity))]
-    [ProtoInclude(30, typeof(PlayerWorldEntity))]
-    [ProtoInclude(40, typeof(VehicleWorldEntity))]
+    [ProtoInclude(51, typeof(PlaceholderGroupWorldEntity))]
+    [ProtoInclude(52, typeof(EscapePodWorldEntity))]
+    [ProtoInclude(53, typeof(PlayerWorldEntity))]
+    [ProtoInclude(54, typeof(VehicleWorldEntity))]
```

Comment:
Not sure to understand

Reply:
This edit makes sense:

https://stackoverflow.com/a/984325/1277156
> [ProtoInclude] works a lot like [XmlInclude] for XmlSerializer - or [KnownType] for DataContractSerializer - it allows it to recognise subclasses of a type during (de)serialization. The only additional thing is that it needs a tag (number) to identify each sub-type (that must be unique, and not clash with any of the fields from the parent type).

In other words, `ProtoInclude` is like a `[DataMember(Order = x)]` in that you don't need gaps in between includes. **But** they should be unique in the same type. Using a starting offset of 50 for protoincludes will help not conflict with the `[DataMember(Order = x)]` on the fields *and* still leave room for more fields without becoming binary incompatible.

Reply:
From the docs I found that the key needs to be unique (scope here being `ProtoInclude` as well as `DataMember`) for the type it's declared in. No docs explained what happens for types which are parents while being a subtype at the same time. My guess is that they will collide. Anyway I changed it to 51+ to better reflect the range we have here. While Entity.cs increments it's `ProtoInclude` key by 10, `WorldEntity` increments now by 1 starting from the Key it has in `Entity` (50).

Reply:
<https://stackoverflow.com/a/56994082/1277156>